examples/noise-rtc/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@waku/noise-example",
+  "private": true,
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
+    "start": "webpack-dev-server"
+  },
+  "dependencies": {
+    "@waku/noise": "https://github.com/waku-org/js-noise.git",
+    "js-waku": "^0.29.0-29436ea",
```

Comment:
ok. we need to update the noise library to latest waku library (`@waku/core`) can you track?

Reply:
Will resolve it here - https://github.com/waku-org/js-noise/issues/19